app/javascript/readingList/readingList.jsx

```diff
@@ -210,30 +217,65 @@ export class ReadingList extends Component {
               placeholder="Search..."
               className="crayons-textfield"
             />
-          </div>
-        </header>
-
-        <div className="crayons-layout crayons-layout--2-cols">
-          <ItemListTags
-            availableTags={availableTags}
-            selectedTags={selectedTags}
-            onClick={this.toggleTag}
-          />
-
-          <main className="crayons-layout__content" id="main-content">
-            <div className="crayons-card mb-4">
-              {items.length > 0 ? itemsToRender : this.renderEmptyItems()}
-            </div>
-
-            <ItemListLoadMoreButton
-              show={showLoadMoreButton}
-              onClick={this.loadNextPage}
+            <MediaQuery
+              query={`(width <= ${BREAKPOINTS.Medium - 1}px)`}
+              render={(matches) => {
+                return (
+                  matches && (
+                    <TagList
+                      availableTags={availableTags}
+                      selectedTag={selectedTag}
+                      onSelectTag={this.toggleTag}
+                      isMobile={true}
+                    />
+                  )
+                );
+              }}
             />
-          </main>
-
-          {snackBar}
-        </div>
-      </section>
+          </fieldset>
+        </header>
+        <MediaQuery
+          query={`(width >= ${BREAKPOINTS.Medium}px)`}
+          render={(matches) => {
```

Comment:
@aitchiss as [per your question in the PR for creating the `<MediaQuery />`](https://github.com/forem/forem/pull/12809#discussion_r582669298) component, this is why I opted for a render prop.